### PR TITLE
Es6

### PR DESCRIPTION
**Pull request by franclobo**
### Awesome books ES6
- Re-create my Awesome books app following the rules:

1. Use ES6 syntax.
2. Divide the code into modules.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+
+import { books, addNew, contact, titulo, autor} from "./modules/variables.js"
+import { containerBooks } from "./modules/list.js"
+import Libro from "./modules/list.js"
+import startTime from "./modules/clock.js";
```
Comment (phelian23) [inline index.js]: - [x] I can see that you are trying to import some variables and the startTime form variables and clock js files, however, the variables.js and clock.js files are not present in your repository. Kindly add them

File: index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  containerBooks.style.display = 'none';
+  addNew.style.display = 'flex';
+  contact.style.display = 'none';
+});
+
+contactBtn.addEventListener('click', () => {
+  containerBooks.style.display = 'none';
+  addNew.style.display = 'none';
+  contact.style.display = 'flex';
+});
```
Comment (phelian23) [inline index.js]: - [x] The page navigations are not working because the variables called are not present. Kindly fix this.

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @franclobo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Linters are passing
✅ Descriptive pull request
✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (franclobo) [reply index.js]: Added clock.js and variables.js to repository.

File: index.js (hunk 2 above)
Comment (franclobo) [reply index.js]: Fixed.

**Review by smunozmo — APPROVED** [review]
Hi @franclobo, 👋 

## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**